Publish engineering principles (#99)
ChainSafe/engineering-handbook: docs/1_principles/index.md

---
sidebar_position: 3
sidebar_label: Principles
---

# Engineering Principles

> <p class="page-caption">The following principles should provide a solid base for all engineers in ChainSafe to act, help them make decisions and navigate their autonomy.</p>

Reviewer: ```suggestion
<p class="page-caption">The following principles should provide a solid base for all engineers at ChainSafe to act, make decisions, and navigate their autonomy.</p>
```
